api/src/main/java/uk/gov/dvsa/mot/trade/api/RfrAndAdvisoryItem.java

```diff
@@ -30,4 +31,15 @@ public RfrAndAdvisoryItem setType(String type) {
         this.type = type;
         return this;
     }
+
+    public Boolean getDangerous() {
```

Comment:
Getter for booleans should follow this pattern: `is<booleanName>()`.

Also, do you need to use the `Boolean` class instead of the `bool` primitive?

Rename to `public bool isDangerous() {...}` please